pkg/apis/core/validation/project.go

```diff
@@ -15,10 +15,13 @@
 package validation
 
 import (
+	"fmt"
 	"strings"
 
 	"github.com/gardener/gardener/pkg/apis/core"
 
+	"github.com/gardener/gardener/pkg/apis/core/v1alpha1/constants"
```

Comment:
```suggestion
	"github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
```

Though, probably it's better to add the `GardenerSeedLeaseNamespace` constant to `pkg/apis/core` as well (and move it from `pkg/apis/core/{v1alpha1,v1beta1/constants` to `pkg/apis/core/{v1alpha1,v1beta1}`).